### ISSUE TITLE
0^0 DomainError

### ISSUE THREAD

**Issue by fgerick** [JuliaMath/DoubleFloats.jl#82]
Is there a reason why DoubleFloats.jl doesn't use the same convention as Base Julia? As far as I know the other Base number types in Julia return ones instead of throwing a DomainError, so why would DoubleFloats do so?

```julia
function Base.:(^)(r::DoubleFloat{T}, n::Int) where {T<:IEEEFloat}
    if (n == 0)
        iszero(r) && throw(DomainError("0^0"))
        return one(DoubleFloat{T})
    end
...
```

**Comment by JeffreySarnoff** [comment]
reasonable -- will fix

**Comment by JeffreySarnoff** [comment]
fixed in pending merge (v0.9.6)
